.gitignore

```diff
@@ -7,4 +7,5 @@ prebuilds/
 lib/
 libexec/
 share/
+include/
```

Comment:
@piranna Yes, we have plans to introduce a configuration file but sadly, I can't provide you with an ETA yet.

Currently, the next best thing is to use a shell script as part of your repository. So instead of adding build commands in Project Settings, you can create script containing all the commands, and then execute this script in a build command with a single line. This script can of course, include anything from configuration changes to dependency installations.